locale-fork/locale/de/LC_MESSAGES/djangojs.po

```diff
@@ -910,68 +925,207 @@ msgstr ""
 msgctxt "kosmo"
-msgid " Unarchive"
-msgstr ""
+msgid "Last edited on "
+msgstr "Zuletzt bearbeitet am"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:228
+msgctxt "kosmo"
+msgid "Created on "
+msgstr "verfasst am"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:7
+msgctxt "kosmo"
+msgid "Block"
+msgstr "Blockieren"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:8
+msgctxt "kosmo"
+msgid "Unblock"
+msgstr "Entblocken"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:9
+msgctxt "kosmo"
+msgid "Blocked"
+msgstr "Blockiert"
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:167
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:10
+msgctxt "kosmo"
+msgid "Add statement"
+msgstr "Mitteilung hinzufügen"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:11
 msgctxt "kosmo"
 msgid "This negative comment was blocked because it is spam"
-msgstr ""
+msgstr "Dieser Beitrag wurde blockiert, da es sich um Spam handelt."
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:12
+msgctxt "kosmo"
+msgid "Highlight"
+msgstr "Hervorheben"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:13
+msgctxt "kosmo"
+msgid "Unhighlight"
+msgstr "Hervorhebung aufheben"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:14
+msgctxt "kosmo"
+msgid "Highlighted"
+msgstr "Hervorgehoben"
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:103
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:15
+msgctxt "kosmo"
+msgid "Unarchive"
+msgstr "Archivierung aufheben"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:16
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:10
+msgctxt "kosmo"
+msgid "Archived"
+msgstr "Archiviert"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx:17
+msgctxt "kosmo"
+msgid "Edit statement"
+msgstr "Mitteilung bearbeiten"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:9
+msgctxt "kosmo"
+msgid "Pending"
+msgstr "Ausstehend"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:11
+msgctxt "kosmo"
+msgid "View all"
+msgstr "Alle ansehen"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:15
+msgctxt "kosmo"
+msgid "Offensive"
+msgstr "beleidigend"
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:16
+msgctxt "kosmo"
+msgid "Engaging"
+msgstr "Hochwertig"
```

Comment:
lower case?

Reply:
Yes, either "beleidigend" in upper case or "Hochwertig" in lower. Would put beleidigend in upper as English is also in upper.